src/scripts/components/audio/SingleAudioDropzone.jsx

```diff
@@ -211,7 +211,8 @@ export default class SingleAudioDropzone extends React.Component {
                   className="progress-container__bar"
                   style={progressBarStyle}
                 >
-                  Loading ({fileStatus.progress}%)
+                  Loading ({fileStatus.progress}
+                  %)
```

Comment:
Is this newline intentional?

Reply:
I don't know why, but the lintter, suggested that.